README.md

```diff
@@ -90,6 +90,7 @@ The ON/OFF depends on what kind of protobuf library you have. Shared libraries a
 If you are building ONNX from source, it is recommended that you also build Protobuf locally as a static library. The version distributed with conda-forge is a DLL, but ONNX expects it to be a static library. Building protobuf locally also lets you control the version of protobuf. The tested and recommended version is 3.16.0.
 
 The instructions in this README assume you are using Visual Studio.  It is recommended that you run all the commands from a shell started from "x64 Native Tools Command Prompt for VS 2019" and keep the build system generator for cmake (e.g., cmake -G "Visual Studio 16 2019") consistent while building protobuf as well as ONNX.
+If you have any protobuf shared libraries (such as libprotobuf) installed, and you are planning to build Protobuf locally as a static library, ensure that the libprotobuf version is also set to 3.16.0 to ensure compatibility.
```

Comment:
Ideally, it would be the best to only have one Protobuf installed in the environment to prevent any possible conflict issues. IMO, perhaps mention something like: If you see Protobuf failures while building Protobuf or ONNX from source, please make sure you only have one installed Protobuf (either static or shared) with recommended version to ensure compatibility.